infra/terraform/global/iam.tf

```diff
@@ -25,3 +25,76 @@ EOF
 resource "aws_iam_access_key" "auth0_ses" {
     user = "${aws_iam_user.auth0_ses.name}"
 }
+
+resource "aws_iam_role" "softnas" {
```

Comment:
Unfortunately we can't - SoftNAS looks for this policy by name, so it has to be shared across environments.